source/linux/Foundational_Components/Kernel/Kernel_Drivers/Display/DSS7.rst

```diff
@@ -362,6 +362,9 @@ Device Tree Node
 Documentation for tidss device tree node and its properties can be found in linux kernel device tree bindings in below directory
 ``Documentation/devicetree/bindings/display/ti/``. Seperate binding files are present for different version of the DSS controller.
 
+.. ifconfig:: CONFIG_part_variant in ('J721S2')
+
+   - Pin designated as HPD for DP should not be used for other purpose, irrespective of whether it is connected as HPD or not.
```

Comment:
is this covered in TRM? if so, point out chapter. DO NOT make s/w SDK documentation the goto place for careabouts of the SoC.

Reply:
@h-shenoy could you verify this and resolve this conversation if it is actually covered in the TRM?

Reply:
No, I did not find information regarding HPD pin connection status in TRM , let me know if information regarding the can be made part of SDK ?

Reply:
ideally, you wont document SoC features in S/w documentation - you should file a internal ticket asking it to be documented in TRM and discuss with applications team how to handle it best.

Reply:
@uditkumarti , should this be documented in TRM and not in SDK docs as mentioned in ticket?

Reply:
@uditkumarti

Reply:
needs to be part of TRM . Please reject this PR